exporting/prometheus/remote_write/README.md

```diff
@@ -0,0 +1,38 @@
+# Prometheus remote write exporting connector
+
+## Prerequisites
+
+To use the Prometheus remote write API with [storage
+providers](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage), install
+[protobuf](https://developers.google.com/protocol-buffers/) and [snappy](https://github.com/google/snappy) libraries.
+Next, re-install Netdata from the source, which detects that the required libraries and
+utilities are now available.
+
+## Configuration
+
+To enable data sending to a storage provider using the Prometheus remote write API, run `./edit-config exporting.conf`
```

Comment:
We should stick with using the word "exporting" when talking about these connectors.

```suggestion
To enable data exporting to a storage provider using the Prometheus remote write API, run `./edit-config exporting.conf`
```